package-lock.json

```diff
@@ -101,6 +101,12 @@
         "vitest": "0.31.4"
       }
     },
+    "node_modules/@adobe/css-tools": {
+      "version": "4.2.0",
+      "resolved": "https://registry.npmjs.org/@adobe/css-tools/-/css-tools-4.2.0.tgz",
+      "integrity": "sha512-E09FiIft46CmH5Qnjb0wsW54/YQd69LsxeKUOWawmws1XWvyFGURnAChH0mlr7YPFR1ofwvUQfcL0J3lMxXqPA==",
+      "dev": true
+    },
     "node_modules/@airgap/beacon-blockchain-substrate": {
       "version": "3.3.4",
```

Comment:
This is pretty outdated: https://www.npmjs.com/package/@airgap/beacon-blockchain-substrate